bin/zowe-configure-component.sh

```diff
@@ -87,6 +88,64 @@ enable_component(){
     fi
 }
 
+install_zis_plugin() {
+    iterator_index=0
+    ZIS_PATH_DIR=$(read_component_manifest "${TARGET_DIR}/${component_name}" ".zisPlugins[${iterator_index}].path" 2>/dev/null)
+    ZIS_INSTALL_DIR="${TARGET_DIR}/zss/bin"
+    ZIS_INSTALL_FILE="${ZIS_INSTALL_DIR}/zis-plugin-install.sh"
+
+    while [ "${ZIS_PATH_DIR}" != "null" ] && [ -n "${ZIS_PATH_DIR}" ]; do
+
+      if [ -e "${INSTANCE_DIR}/instance.env" ]; then
+          if [ -n "$(read_zowe_instance_variable "ZWES_ZIS_PARMLIB")" ]; then
+              export ZIS_PARMLIB=$(read_zowe_instance_variable "ZWES_ZIS_PARMLIB")
+          fi
+          if [ -n "$(read_zowe_instance_variable "ZWES_ZIS_PLUGINLIB")" ]; then
+              export ZIS_PLUGINLIB=$(read_zowe_instance_variable "ZWES_ZIS_PLUGINLIB")
+          fi
+      fi
+
+      if [ -n "${ZWES_ZIS_PARMLIB}" ]; then #Checks existing env variables
+          export ZIS_PARMLIB="${ZWES_ZIS_PARMLIB}"
+      fi
+          
+      if [ -n "${ZWES_ZIS_PLUGINLIB}" ]; then
+          export ZIS_PLUGINLIB="${ZWES_ZIS_PLUGINLIB}"
+      fi
+
+      if [[ -z "${ZIS_PARMLIB}" ]]; then
+          log_message "ZIS_PARMLIB N/A from env or instance. Skipping ZIS plugin check."
+      else
+          if [[ -z "${ZIS_PLUGINLIB}" ]]; then
+              log_message "ZIS_PLUGINLIB N/A from env or instance. Skipping ZIS plugin check."
+          else
+              log_message "Received ZIS_PARMLIB: ${ZIS_PARMLIB}"
+              log_message "Received ZIS_PLUGINLIB: ${ZIS_PLUGINLIB}"
+              if test -f "$ZIS_INSTALL_FILE"; then
+                  if [ -n "${ZIS_PATH_DIR}" ]; then 
+                      LOADLIB=$(read_component_manifest "${TARGET_DIR}/${component_name}" ".zisPlugins[${iterator_index}].loadlib" 2>/dev/null)
+                      if [ "${LOADLIB}" = "null" ] || [ -z "${LOADLIB}" ]; then # Check custom defined lib directories in manifest.yaml
+                          LOADLIB="loadlib" # default
+                      fi
+                      SAMPLIB=$(read_component_manifest "${TARGET_DIR}/${component_name}" ".zisPlugins[${iterator_index}].samplib" 2>/dev/null)
+                      if [ "${SAMPLIB}" = "null" ] || [ -z "${SAMPLIB}" ]; then
+                          SAMPLIB="samplib" #default
+                      fi
```

Comment:
this is fine, i guess, but doesn't seem to match the documentation you wrote to accompany.

https://github.com/zowe/zss/blob/staging/bin/zis-plugin-install.sh
that script appears to take whatever directories given for loadlib and samplib and handle all files within.

https://github.com/zowe/docs-site/pull/1729/files#diff-388b86598a43590a411488b297bc9583b1fcf194eb552bca1934351c23852dd7R88
this doc implies the purpose of the component specification for loadlib/samplib is to deal with individual files within the (hardcoded name) directories

so these are in conflict: one says files but the other is specifying a folder.
you have 2 possible fixes

1) change the script to allow individual file handling within the folder (potentially useful)
2) remove this logic and documentation about specification of `.zisPlugins[n].samplib` & loadlib since allowing people to vary the folder names rather than standardize them just seems like unnecessary complexity and a source of confusion.

Reply:
Good comments, will do #2 for now and maybe #1 in the near future or if requested